portfolio/src/main/webapp/script.js

```diff
@@ -82,16 +82,11 @@ function fetchBlobstoreUrlAndShowForm() {
 }
 
 function showPostForm() {
-  fetch('/login').then(response => response.text()).then((isLoggedIn) => {
+  fetch('/login_status').then(response => response.json()).then((isLoggedIn) => {
     if (isLoggedIn) {
       console.log("inside IF")
```

Comment:
Nit: if this is only used during your debugging, it may be good to just remove them when you try check in code.

Reply:
We usually only keep the critical debugging print statements in the codebase.

Reply:
My bad! I deleted them after commiting